octodns/provider/constellix.py

```diff
@@ -156,9 +169,108 @@ def record_delete(self, zone_name, record_type, record_id):
             record_type = 'ANAME'
 
         zone_id = self.domains.get(zone_name, False)
-        path = '/{}/records/{}/{}'.format(zone_id, record_type, record_id)
+        path = f'/domains/{zone_id}/records/{record_type}/{record_id}'
         self._request('DELETE', path)
 
+    def pools(self, pool_type):
+        if self._pools is None:
+            self._pools = {}
+            path = f'/pools/{pool_type}'
```

Comment:
Are there multiple values for `pool_type`? If so this only seems to cache the first one that comes through and the others will just get back whatever `pool_type` was passed the first time. (noticed this when I looked back at the original discussion, https://github.com/octodns/octodns/pull/616/files#r505936387.

If there's only one type then it probably makes sense to remove the parameter and hard code, or if it stays around a comment about how there's really only one value would prevent future questions. 

Sorry I didn't catch this on the previous round.

Reply:
@ross , you are right, missed this issue. Thanks for pointing me into it.
Have updated pools cash by adding separate cash values for every supported pool_type: A, AAAA, CNAME

Reply:
Fwiw it's best not to force push once you've opened the PR (as ready for review.) It looses the context and makes it tough to see what's changed since the last round of discussion.

Reply:
Got it, sorry.